### PR TITLE
Add detailed signalR logs

### LINKED ISSUE #1563: Set up server side SignalR log
**Describe the new feature you would like to see**
When a message is sent from the server using signalR, the message should be logged.

**Describe the solution you'd like**
See: https://learn.microsoft.com/en-us/aspnet/core/signalr/diagnostics?view=aspnetcore-8.0

**How will this feature affect the current Threat Model?**
Should not be affected if we treat it as our other server logs.

### PR DESCRIPTION
**Pull request by andchiind**
Closes #1563

### CODE REVIEW

**Review by mrica-equinor — APPROVED** [review]
LGTM :)

File: backend/api/appsettings.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
     "ApplicationInsights": {
       "LogLevel": {
         "Default": "Warning",
-        "Api": "Information"
+        "Microsoft.AspNetCore.SignalR": "Debug",
+        "Microsoft.AspNetCore.Http.Connections": "Debug"
```
Comment (Christdej) [inline backend/api/appsettings.json:28]: Do we want Debug to be the general level or is this something we set when we want to debug something?

Comment (andchiind) [reply backend/api/appsettings.json:28]: To be honest, the documentation is somewhat vague on what will be logged: https://learn.microsoft.com/en-us/aspnet/core/signalr/diagnostics?view=aspnetcore-8.0#server-side-logging
Here they only mention two options, debug and trace, and they state trace is more detailed than debug. As logn as we can see when messages are being sent then I think the log could be useful, but if this log is not detailed enough then another option would be to just use the normal logger in the SignalRService class, though this is theoretically not as reliable as we cannot guarantee that the messages are being sent properly when we call the function.

Comment (Christdej) [reply backend/api/appsettings.json:28]: Lets try with Debug then, and change it if it floods the logs with to much information

**Review by Christdej — APPROVED** [review]

**Pull request merged**